### PR TITLE
Hyacinth 858 - fix and test identifier UI search

### PR DESCRIPTION
**Pull request by elohanlon**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -57,7 +57,7 @@ group :development do
 end
 
 group :test do
-  gem 'capybara', '>= 2.15'
+  gem 'capybara', '~> 3.35'
```
Comment (elohanlon) [inline Gemfile:60]: We were already on capybara 3.35.3 (which is the latest version, so I updated the version here to make it clear that we're not on 2.x).

**Comment by barmintor** [comment]
I'm going to review first thing Wednesday morning. Saw that tests failed - I am restarting in case it was a transient thing.

**Comment by barmintor** [comment]
ok, see it failed again - likely suspect is the chrome install in this PR, right? Can we separate that into a different PR?

**Comment by elohanlon** [comment]
@barmintor It was failing before, so the chrome install was an attempt to fix the failure (but no luck).  Tests pass locally, but not on GitHub Actions CI.  I'll try some other things and see if I can fix the CI run.

**Comment by elohanlon** [comment]
@barmintor I changed the :system tests back to :feature tests and everything passes, so for the sake of development speed I'll keep them that way for now!  Feel free to review whenever is convenient!

File: app/javascript/src/hyacinth_v1/components/digital_objects/search/QueryForm.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import SelectInput from '../../shared/forms/inputs/SelectInput';
 
 import SearchButton from '../../shared/buttons/SearchButton';
 
-const searchTypes = ['KEYWORD', 'TITLE', 'IDENTIFER'];
+const searchTypes = ['KEYWORD', 'TITLE', 'IDENTIFIER'];
```
Comment (barmintor) [inline app/javascript/src/hyacinth_v1/components/digital_objects/search/QueryForm.js:10]: Well that's unfortunate.

**Review by barmintor — APPROVED** [review]
There are a couple of things here that I recommend doing in separate commits in the future (moving an unrelated test to a better location, improving a log message), as the commit history for the file will itself kind of violate the principle of least surprise - but this looks good to me.

**Comment by elohanlon** [comment]
@barmintor Good suggestion!  I just pushed an update to split the changes up into multiple commits.

**Pull request merged**